docs/open3d_ml.rst

```diff
@@ -5,6 +5,7 @@ Open3D-ML
 
 .. image:: https://raw.githubusercontent.com/isl-org/Open3D-ML/master/docs/images/getting_started_ml_visualizer.gif
     :width: 480px
+    :align: center
```

Comment:
Wanted to highlight that since furo is center-aligned theme, unlike sphinx, its better to go for center alignment for images. (IMO, although correct me if you have a contradicting thoughts).